relengapi/blueprints/slaveloan/bugzilla.py

```diff
@@ -0,0 +1,142 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# XXX Much of this functionality should probably be its own
+#     Separate relengapi blueprint
+
+import logging
+
+from bzrest.client import BugzillaClient
+from flask import current_app
+from flask import url_for
+
+log = logging.getLogger(__name__)
+
+MAX_ALIAS = 15
+DEFAULT_BUGZILLA_URL = "https://bugzilla-dev.allizom.org/rest/"
+
+
+def _bzclient():
+    if getattr(current_app, 'bzclient', None):
```

Comment:
Rather than adding this during the first request to use it, it'd be better to add it in an `init_app` function.  Then any config errors are detected at startup, rather than later.

See https://github.com/mozilla/build-relengapi/blob/master/relengapi/blueprints/tokenauth/__init__.py#L339 for an example of such a thing.